docs/blog.md

```diff
@@ -34,26 +34,26 @@ Since keyboard users pretty much can't do anything without knowing where focus i
 That should make everyone happy, right?  Unfortunately, not really.
 
 ## The Fly in the Ointment
-If you've never done so, try adding a global focus rule like the one above and use the UI for a bit.  If you're using a mouse, it can be pretty disconcerting and unpleasant. In fact, it can be downright confusing as suddenly you begin to notice that focus ring in ways/places you probably didn't before you added it. Understanding why this happens is where things get _really_ interesting. There's a fly in the ointment here, but where?
+If you've never done so, try adding a global focus rule like the one above and use the UI for a bit.  If you're using a mouse, it can be pretty disconcerting and unpleasant. In fact, it can be downright confusing as suddenly you begin to notice that focus ring in ways/places you probably didn't before you added it. Understanding why this happens is where things get _really_ interesting. Clearly there's a fly in the ointment here, but where?
  
-To answer this, try clicking plain old (unstyled) `<button>` element with a mouse. Since you're looking for it, you'd probably now notice that it did not get the focus ring. However, if you reached it by way of keyboard navigation, it would. But the weird thing is, if you've _set_ a focus style on it, that focus style will be applied _regardless_ of how you used it.  
+To answer this, try clicking plain old (unstyled) `<button>` element with a mouse. Since you're paying close attention to it now, you'd probably notice that it did not get the focus ring. However, if you reached it by way of keyboard navigation, it would. But the weird thing is, if you've _set_ a focus style on it, that focus style will be applied _regardless_ of how you used it, and this leads to all sorts of unfortunate consequences (like the preponderance of sites removing it altogether).  
```

Comment:
On Sat, Jul 25, 2015 at 1:31 PM, Alice <notifications@github.com> wrote:

> In docs/blog.md
> <https://github.com/bkardell/modality/pull/4#discussion_r35484951>:
>
> >
> > -To answer this, try clicking plain old (unstyled) `<button>` element with a mouse. Since you're looking for it, you'd probably now notice that it did not get the focus ring. However, if you reached it by way of keyboard navigation, it would. But the weird thing is, if you've _set_ a focus style on it, that focus style will be applied _regardless_ of how you used it.
> > +To answer this, try clicking plain old (unstyled) `<button>` element with a mouse. Since you're paying close attention to it now, you'd probably notice that it did not get the focus ring. However, if you reached it by way of keyboard navigation, it would. But the weird thing is, if you've _set_ a focus style on it, that focus style will be applied _regardless_ of how you used it, and this leads to all sorts of unfortunate consequences (like the preponderance of sites removing it altogether).
>
> Perhaps move this last clause down to the next paragraph, where we're
> talking about the weirdness of the focus ring.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bkardell/modality/pull/4/files#r35484951>.
>

I'm not personally bothered by it, but have at it.


-- 
Brian Kardell :: @briankardell :: hitchjs.com